### ISSUE TITLE
Group assigning is not working

### ISSUE THREAD

**Issue by ostasevych** [nextcloud/end_to_end_encryption#422]
Cannot assign the group which may use the E2EE functionality from the web UI.
I am using Nextcloud 26 with the end-to-end encryption app installed. 
I would like to limit the access to the encryption to a certain group, eg admin.
For that I am going to the Security setting in the Settings. There I pick up a group, click Save. And when I reload the page I see only the button Save without understanding which group has access to the E2EE functionality.
![image](https://user-images.githubusercontent.com/5199030/233381467-203a8c46-10e4-4f47-9f5f-8fceba7348b8.png)
When I click Save again and reload the page I see that none group is selected.
![image](https://user-images.githubusercontent.com/5199030/233381647-a7274928-2653-46c7-80cb-6285a8e84e8d.png).
When I select again the group and click Save it, reload the page I see strange settings.
![image](https://user-images.githubusercontent.com/5199030/233380657-5d4f6273-f30c-4333-98a9-35ab93106977.png)
Clicking in the field I see that any group is selected.
![image](https://user-images.githubusercontent.com/5199030/233381986-253a2e68-1015-41cb-a10f-7e34fff9f045.png)
So, I assume, that this functionality is not working at all.

**Comment by HuYlllc** [comment]
I also encountered the same issue in NC 26.0.0, and then I found that the group data was already in the allowed_groups of E2EE under the oc_appconfig table in the database.
<img width="750" alt="image" src="https://user-images.githubusercontent.com/46414020/234236012-4ecd3679-3fca-441c-88d2-6d2c44ffec29.png">
When I switched to the personal security interface, I found that the E2EE function for users in the admin group was unavailable.
<img width="1051" alt="image" src="https://user-images.githubusercontent.com/46414020/234237273-b9e9d445-0b4f-4af5-a4bc-cf94646965dd.png">
I think that although the content of the database is correct, the E2EE did not take effect for user groups in the backend logic.
